lib/ansible/modules/windows/win_service.ps1

```diff
@@ -59,6 +61,65 @@ if ($null -ne $path) {
     $path = [System.Environment]::ExpandEnvironmentVariables($path)
 }
 
+Function ReversedHexValue ($v) {
+
+  $res = (("{0:x8}" -f $v) -split '(..)' | Where-Object { $_ }).split(" ")
+
+  [array]::Reverse($res)
+
+  $res = [system.string]::Join('', $res)
+
+  $res
+}
+
+Function DecValue ($v) {
+
+  $res = ($v -split '(..)' | Where-Object { $_ }).split(" ")
+
+  [array]::Reverse($res)
+
+  $res = [system.string]::Join('', $res)
+
+  $res = [convert]::toint32($res, 16)
+
+  $res
+}
+
+Function RecoveryActionMapping ($a) {
```

Comment:
Why not using a hashtable?

Reply:
And then I do lookups in that hashtable with the value I want to map?

Reply:
yes

Reply:
I tested the hashtable option and couldn't make it work.